modules/web/js/ballerina/utils/ace-formatter.js

```diff
@@ -258,12 +258,14 @@ define(function (require) {
                             }
                             if(!skipBreakAfter){
                                 value += newLine;
+                                var indent = indentation;
                                 // indent
-                                if(token.type !== 'paren.lparen' || token.value !== '{' || nextToken.type !== 'paren.rparen' || nextToken.value !== '}') {
-                                    // if block has no content don't indent so the closing '{' is not unnecessarily indented
-                                    for (var i = 0; i < indentation; i++) {
-                                        value += tab;
-                                    }
+                                if(token.type === 'paren.lparen' && token.value === '{' && nextToken.type === 'paren.rparen' && nextToken.value === '}') {
+                                    // if block has no content don't indent so the closing '}' is not unnecessarily indented
+                                    indent -= 1;
+                                }
+                                for (var i = 0; i < indent; i++) {
+                                    value += tab;
```

Comment:
If the value of "indent" cannot be a minus value, we can write this as shown below avoiding the for loop.

value += (tab* indent);